packages/pages/src/vite-plugin/build/build.ts

```diff
@@ -35,6 +36,11 @@ export const build = (projectStructure: ProjectStructure): Plugin => {
               assetFileNames: "assets/static/[name]-[hash][extname]",
               chunkFileNames: "assets/static/[name]-[hash].js",
             },
+            plugins: [
+              externalGlobals({
+                "cross-fetch": "fetch",
```

Comment:
I don't think we should do this. It's an unclear dependency and surprising/atypical behavior. 

It's odd that we'd bundle cross-fetch dependencies specifically to the fetch global due to 
 our dependency on deno.

Instead of doing this why don't we provide a a mechanism natively through our package that allows users to fetch across all environments that we support. 

```js
const fetch = (// fetchAPI) => {
  if node {
    // cross-fetch or node-fetch
  }
  
  // otherwise we're deno
  fetch(fetchApi)
}
```

Reply:
probably: https://www.npmjs.com/package/node-fetch

Reply:
Updated to export some new util functions instead.